package.json

```diff
@@ -61,10 +65,19 @@
     "eslint": "^3.3.1",
     "eslint-config-babel": "^1.0.1",
     "eslint-plugin-babel": "^3.3.0",
-    "eslint-plugin-flow-vars": "^0.5.0"
+    "eslint-plugin-flow-vars": "^0.5.0",
+    "mocha": "^3.0.2"
   },
   "babel": {
     "plugins": [
+      "babel-plugin-transform-es2015-destructuring",
```

Comment:
Humm, good to know. Well, this very plugin would be useful here so until we can just use `./lib` in place of this whole list lets pull transformations in as needed. Once we're comfortable with where everything is I'll go back over and pull in extra transformations for Node 0.10.